glean-core/rlb/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-name = "glean-preview"
+name = "glean"
 version = "0.0.5"
```

Comment:
As said above, do we immediately bump this to the same version as glean-core (33.y.z)?